cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -183,6 +186,27 @@ var CMS = window.CMS || {};
                     );
                 });
 
+                // prevents single click from messing up the edit call
+                this.ui.container.find('a').on('click', function (e) {
+                    if (e.shiftKey || e.ctrlKey || e.metaKey) {
+                        return;
+                    }
+
+                    e.preventDefault();
+                    e.stopImmediatePropagation();
```

Comment:
I've done it intentionally to avoid custom js, but yes, it makes sense to let it run, though. Removed.